src/mappings/mappingHandlers.ts

```diff
@@ -167,6 +168,7 @@ export async function handleEvent(event: SubstrateEvent): Promise<void> {
       moduleId: moduleId as ModuleIdEnum,
       params: args,
       event,
+      specVersion,
```

Comment:
nit -> Was there a need to pass this as handlerArgs? we could have got it from event itself (event.block.specVersion)? I don't have problem if you leave it as it as

Reply:
I didn't realize I had a connection to the block. Will change